### ISSUE TITLE
Following the "Minimal Setup" in the README yields ConfigError: Config "vue/base/setup": Key "plugins": Cannot redefine plugin "vue".

### ISSUE THREAD

**Issue by brianreeve** [vuejs/eslint-config-typescript#161]
The minimal setup as found in the README causes an error in VSCode's "ESLint: Show Output Channel" console whenever I open a new file or navigate to another open editor tab.

```
[Error - 1:29:23 PM] An unexpected error occurred:
[Error - 1:29:23 PM] ConfigError: Config "vue/base/setup": Key "plugins": Cannot redefine plugin "vue".
    at rethrowConfigError (/Users/brian/projects/REDACTED/REDACTED/node_modules/@eslint/config-array/dist/cjs/index.cjs:328:8)
    at /Users/brian/projects/REDACTED/REDACTED/node_modules/@eslint/config-array/dist/cjs/index.cjs:1174:5
    at Array.reduce (<anonymous>)
    at FlatConfigArray.getConfigWithStatus (/Users/brian/projects/REDACTED/REDACTED/node_modules/@eslint/config-array/dist/cjs/index.cjs:1167:43)
    at FlatConfigArray.getConfigStatus (/Users/brian/projects/REDACTED/REDACTED/node_modules/@eslint/config-array/dist/cjs/index.cjs:1209:15)
    at ESLint.lintText (/Users/brian/projects/REDACTED/REDACTED/node_modules/eslint/lib/eslint/eslint.js:957:13)
    at async /Users/brian/.vscode/extensions/dbaeumer.vscode-eslint-3.0.10/server/out/eslintServer.js:1:26981
    at async M (/Users/brian/.vscode/extensions/dbaeumer.vscode-eslint-3.0.10/server/out/eslintServer.js:1:19807)
    at async /Users/brian/.vscode/extensions/dbaeumer.vscode-eslint-3.0.10/server/out/eslintServer.js:1:234554
    at async /Users/brian/.vscode/extensions/dbaeumer.vscode-eslint-3.0.10/server/out/eslintServer.js:1:63886
```

Removing either `pluginVue.configs['flat/essential']` or `vueTsConfigs.recommended` makes eslint happy, but I want to at least start with both of these to achieve the baseline recommended setup until I understand better what we might want to tweak.

This is in a Vue 3 application workspace in a turborepo. 

Relevant versions are:

```
"@vue/eslint-config-typescript": "^14.5.0",
"eslint": "^9.23.0",
"eslint-plugin-vue": "^10.0.0",
```

Thanks!

**Comment by brianreeve** [comment]
Thank you for the suggestion. I tried that but I still get the same error after restarting the ESLint server.

**Comment by willwillems** [comment]
I fixed this by either getting `vueTsConfigs.recommended.toConfigArray()` instead of just `vueTsConfigs.recommended`, or by doing:
```
const vts = vueTsConfigs.recommended.asExtendedWith(pluginVue.configs['flat/recommended'])
```
and using that object.

No idea which is better, I'm going with the second one.
